app-emulation/vagrant/vagrant-2.2.14.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+USE_RUBY="ruby25 ruby26"
+
+RUBY_FAKEGEM_EXTRADOC="CHANGELOG.md README.md"
+RUBY_FAKEGEM_GEMSPEC="vagrant.gemspec"
+RUBY_FAKEGEM_EXTRAINSTALL="keys plugins templates version.txt"
+RUBY_FAKEGEM_TASK_DOC=""
+
+inherit bash-completion-r1 ruby-fakegem
+
+DESCRIPTION="A tool for building and distributing development environments"
+HOMEPAGE="https://vagrantup.com/"
+SRC_URI="https://github.com/hashicorp/vagrant/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+virtualbox test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="${RDEPEND}
+	app-arch/libarchive
+	net-misc/curl
+	virtualbox? ( || ( app-emulation/virtualbox ) )"
```

Comment:
This is a bit weird. To me looks like this USE flag has no function in this ebuild.

Reply:
It looks strange but it worked when I tested it, let's change it to a more simple form:
```
	virtualbox? ( app-emulation/virtualbox )"
```

Reply:
But this USE flag has no function in the ebuild - no configure option, nothing in the installed file list changes whether it's on/off. It's just controlling runtime dependency currently, which is prohibited by our QA policy:
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0001

I'd suggest using `optfeature` from *optfeature.eclass* instead.